RELEASENOTES.md

```diff
@@ -1,2 +1,3 @@
 - Fix `gh bbs2gh grant-migrator-role` so it doesn't throw `System.InvalidOperationException`
-- Rename `AWS_ACCESS_KEY` and `AWS_SECRET_KEY` environment variables to `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` respectively to align with the environment variables that the AWS CLI already uses. Old environment variables are still supported but they will be removed in future. 
\ No newline at end of file
+- Rename `AWS_ACCESS_KEY` and `AWS_SECRET_KEY` environment variables to `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` respectively to align with the environment variables that the AWS CLI already uses. Old environment variables are still supported but they will be removed in future. 
+- Add handling of files larger than 2GB
```

Comment:
```suggestion
- Add support for migration archives larger than 2GB when using the blob storage flow
```

Reply:
Updated in ffeb3b6bf58010bc552e140b2effca460a9f2bff